storage-stackdriver/pom.xml

```diff
@@ -37,22 +37,37 @@
 
     <dependency>
       <groupId>com.linecorp.armeria</groupId>
-      <artifactId>armeria-grpc</artifactId>
-      <!-- Not using armeria BOM as zipkin-server + grpc-google-cloud-trace-v2 disagree on deps -->
-      <version>${armeria.version}</version>
+      <artifactId>armeria-grpc-protocol</artifactId>
     </dependency>
 
     <dependency>
       <groupId>com.google.api.grpc</groupId>
-      <artifactId>grpc-google-cloud-trace-v2</artifactId>
+      <artifactId>proto-google-cloud-trace-v2</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
```

Comment:
/me thinking.. no need to change anything

suppose we need this as long as we aren't using something else to generate the protos for cloud trace (ex the library above). While not required, it would be neat to use wire or something to generate these as they are simple types and the dependency tree is less frantic. That said, I think we'd be in the same boat eventually due to oauth etc, so probably not important.